integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDBOperator.java

```diff
@@ -0,0 +1,740 @@
+    String encryptionSecretName = "encryptionsecret";
+    assertDoesNotThrow(() -> createDomainSecret(encryptionSecretName, "weblogicenc",
+        "weblogicenc", wlsDomainNamespace),
+        String.format("createSecret failed for %s", encryptionSecretName));
+
+    logger.info("Create database secret");
+    final String dbSecretName = wlsDomainUid + "-db-secret";
+
+    cpUrl = "jdbc:oracle:thin:@//" + dbUrl;
+    logger.info("ConnectionPool URL = {0}", cpUrl);
+    assertDoesNotThrow(() -> createDatabaseSecret(dbSecretName,
+        "sys as sysdba", "Oradoc_db1", cpUrl, wlsDomainNamespace),
+        String.format("createSecret failed for %s", dbSecretName));
+    String configMapName = "jdbc-jms-recovery-configmap";
+
+    createConfigMapAndVerify(configMapName, wlsDomainUid, wlsDomainNamespace,
+        Arrays.asList(MODEL_DIR + "/jms.recovery.yaml"));
+
+    // create PV, PVC for logs/data
+    createPV(pvName, wlsDomainUid, ItDBOperator.class.getSimpleName());
+    createPVC(pvName, pvcName, wlsDomainUid, wlsDomainNamespace);
+
+    // create job to change permissions on PV hostPath
+    createJobToChangePermissionsOnPvHostPath(pvName, pvcName, wlsDomainNamespace);
+
+    // create the domain CR with a pre-defined configmap
+    createDomainResourceWithLogHome(wlsDomainUid, wlsDomainNamespace,
+        MII_BASIC_IMAGE_NAME + ":" + MII_BASIC_IMAGE_TAG,
+        adminSecretName, OCIR_SECRET_NAME, encryptionSecretName,
+        replicaCount, pvName, pvcName, "cluster-1", configMapName,
+        dbSecretName, false, false, true);
+
+    // wait for the domain to exist
+    logger.info("Check for domain custom resource in namespace {0}", wlsDomainNamespace);
+    testUntil(domainExists(wlsDomainUid, DOMAIN_VERSION, wlsDomainNamespace),
+        logger,
+        "domain {0} to be created in namespace {1}",
+        wlsDomainUid,
+        wlsDomainNamespace);
+
+    logger.info("Check admin service and pod {0} is created in namespace {1}",
+        wlsAdminServerPodName, wlsDomainNamespace);
+    checkPodReadyAndServiceExists(wlsAdminServerPodName, wlsDomainUid, wlsDomainNamespace);
+    adminSvcExtRouteHost = createRouteForOKD(getExternalServicePodName(wlsAdminServerPodName), wlsDomainNamespace);
+    // create the required leasing table 'ACTIVE' before we start the cluster
+    createLeasingTable(wlsAdminServerPodName, wlsDomainNamespace, dbUrl);
+    // check managed server services and pods are ready
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed server services and pods are created in namespace {0}",
+          wlsDomainNamespace);
+      checkPodReadyAndServiceExists(wlsManagedServerPrefix + i, wlsDomainUid, wlsDomainNamespace);
+    }
+
+    //"Verify JMS/JTA Service migration with File(JDBC) Store"
+    testMiiJmsJtaServiceMigration();
+  }
+
+  /**
+   * Verify JMS/JTA Service is migrated to an available active server.
+   */
+  private void testMiiJmsJtaServiceMigration() {
```

Comment:
Is this a test or helper method? If it is a helper method, can you move this to the end?

Reply:
helper method, moved below Test methods.